frontend/src/app/main/ui/workspace/sidebar/options/menus/component.cljs

```diff
@@ -245,9 +245,8 @@
 
                                   [[(tr "workspace.shape.menu.detach-instance") do-detach-component]
                                    (when can-update-main?
-                                     [:*
-                                      [(tr "workspace.shape.menu.reset-overrides") do-reset-component]
-                                      [(tr "workspace.shape.menu.update-main") do-update-component]])
+                                     [(tr "workspace.shape.menu.reset-overrides") do-reset-component]
```

Comment:
This line is ignored on `when`, this is why the `[:*` is used previosly, to return both items

Reply:
You are right. But I can't use [:* becuase it will be detected as a submenu. I've made an alternative version...